src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+        String city = null;
+        Integer person = null;
+        Integer price = null;
+        Date reservedFrom = null;
+        Date reservedTo = null;
+        try {
+            city = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+            person = (Integer) inputDataFromUser("Enter the number of persons: ", true,
+                    false, false, false);
+            price = (Integer) inputDataFromUser("Enter maximum price rate: ", true,
+                    false, false, false);
+            reservedFrom = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            reservedTo = (Date) (inputDataFromUser("Enter the departure date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            compareDate(reservedFrom, reservedTo);
+        } catch (IOException e) {
+            outputDataInConsole("Enter arrival date in the format");
+        }
+        Map<String, List<Room>> foundRooms = hotelController.getAvailableRooms(city, person, price,
+                reservedFrom, reservedTo);
+        if (foundRooms.values().isEmpty()) {
+            outputDataInConsole("Room with such parameters not found");
+        } else {
+            outputDataInConsole("The list of found rooms:\n ");
+            outputListInConsole(foundRooms.entrySet());
+            outputSplitLine();
+            bookRoomMenu(reservedFrom, reservedTo, foundRooms);
+            outputDataInConsole("For room reservation please fill out the following form");
+        }
+        return foundRooms;
+    }
+
+    private void bookRoomMenu(Date reservedFrom, Date reservedTo, Map<String, List<Room>> foundRooms) throws IOException {
+        long reserveId = 0L;
+        boolean bookFlaf = true;
+        int numberHotel;
+        Hotel hotelSet = null;
+        Room roomSet = null;
+        while (bookFlaf) {
+            outputDataInConsole("Do you have booking room?");
+            showUserMenu("Booking menu");
+            Integer bookRoom = (Integer) inputDataFromUser("Enter number: ", true,
+                    false, false, false);
+            switch (bookRoom) {
+                case (1):
+                    try {
+                        Integer hotelNumber = (Integer) inputDataFromUser("Enter number hotel: ",
+                                true, false, false, false);
+                        Integer roomNumber = (Integer) inputDataFromUser("Enter number room: ",
+                                true, false, false, false);
+                        int countRooms;
+                        for (Map.Entry<String, List<Room>> e : foundRooms.entrySet()) {
+                            for (Room e1 : e.getValue()) {
+
+                            }
+//                                System.out.println(e.getKey() + " = "+ e1.Out());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)